src/js/popup.js

```diff
@@ -172,15 +172,15 @@ async function launchScript(shuffle = false) {
     window.close()
     chrome.tabs.query({active: true, currentWindow: true}, (tabs) => {
         chrome.tabs.sendMessage(tabs[0].id, {
-                type: "close_popup"
-            },
-            (response) => {
-                if (response) {
-                    //console.log(response);
-                }
-            });
-        window.close();
-    });
```

Comment:
why did you removed all `;` if the convetion before was the usage of `;`, it's not usefull.

Reply:
may be my IDE, if it's the convention i'll put them back